install.py

```diff
@@ -607,7 +624,8 @@ def install_udbsync_rootssh():
 
 def install_presenced():
     requires('libprologin')
-    requires('nginxcfg')
+    # FIXME(halfr): remove if proven not useful
```

Comment:
But where? presenced runs on the user machines and afaik there is not need of a nginx server there.